src/ui/ResultLink/ResultLink.ts

```diff
@@ -273,9 +273,10 @@ export class ResultLink extends Component {
   ) {
     super(element, ResultLink.ID, bindings);
 
+    const globalOptions = Component.globallyConfiguredOptions[ResultLink.ID] || {};
```

Comment:
I think the `globalOptions` static object could be removed from `Component.ts`, and instead resolved here with `this.searchInterface.options.originalOptionsObject`

Reply:
This is much better, adjusting!